ovh/resource_cloud_project_gateway.go

```diff
@@ -0,0 +1,265 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/ovh/terraform-provider-ovh/ovh/helpers"
+
+	"github.com/ovh/go-ovh/ovh"
+)
+
+func resourceOvhCloudProjectGatewayImportState(
+	d *schema.ResourceData,
+	meta interface{},
+) ([]*schema.ResourceData, error) {
+	givenId := d.Id()
+	splitId := strings.SplitN(givenId, "/", 2)
+	if len(splitId) != 2 {
+		return nil, fmt.Errorf("Import Id is not OVH_CLOUD_PROJECT/network_id formatted")
+	}
+	d.SetId(splitId[1])
+	d.Set("service_name", splitId[0])
+	results := make([]*schema.ResourceData, 1)
+	results[0] = d
+	return results, nil
+}
+
+func resourceCloudProjectGateway() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudProjectGatewayCreate,
+		Read:   resourceCloudProjectGatewayRead,
+		Update: resourceCloudProjectGatewayUpdate,
+		Delete: resourceCloudProjectGatewayDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceOvhCloudProjectGatewayImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"service_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OVH_CLOUD_PROJECT_SERVICE", nil),
+				Description: "Service name of the resource representing the id of the cloud project.",
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
```

Comment:
even if `forceNew: false` i should remove the line ?

Reply:
my bad, we generally don't use the flag when it is false since it's the default value, I would rather remove the `false` ones for the consistency with the rest of the codebase